fortress-platform/fortress-workflow.xml

```diff
@@ -79,15 +79,18 @@
 
                 <!-- Update the bookmark -->
                 <ParseDate pattern="yyyy-MM-dd'T'HH:mm:ss[.SSS]'Z'" timeZone="${/time_zone}" date="${max(/get_alerts/body/records/createdAt)}" savePath="/last_event_time" />
-            <Log type="info" message="${/last_event_time}"/>
+                <Set path="/lastAlert" value="${max(/get_alerts/body/records/createdAt)}"/>
                 <Set path="/bookmark" value="${/last_event_time + 1}" />
-
             </If>
-    </Actions>	
+            <Else>
+                <Set path="/heartBeatLog" value="The connection to the Fortress API was successfully established; however, there are no new alerts available for retrieval at this moment. The last alert received was at: ${/lastAlert} (UTC)." />
```

Comment:
Is it possible that the `lastAlert` will have have now value?  Why not take `bookmark`? Also you are using `max` twice on `createdAt`

Reply:
the lastAlert can't be new because we added 1 ms for the next run.
Fixed the using max twice.